app/grandchallenge/algorithms/templates/algorithms/algorithm_detail.html

```diff
@@ -206,6 +212,20 @@ <h2>Permission Requests</h2>
 
             </div>
 
+            <div class="tab-pane fade" id="builds" role="tabpanel"
+                 aria-labelledby="v-pills-builds-tab">
+
+                <h2>Builds</h2>
+
+                {% for build in builds %}
+                    <div class="mb-2">
+                        <a class="btn btn-{% if build.status == 'SUCCEEDED' %}success{% elif build.status == 'IN_PROGRESS' %}warning{% else %}danger{% endif %}">
```

Comment:
Don't need to change this but I quite like the pattern of adding `animate` and `status_class` on models to do this, saves some repetition and UI bugs https://github.com/comic/grand-challenge.org/blob/be58bf10a93baa4344f783535e3badd8ce7c9aa5/app/grandchallenge/components/models.py#L844

Reply:
I'll use that pattern in the future, thanks for pointing it out.